Makefile

```diff
@@ -0,0 +1,363 @@
+# using this Go version instead: https://github.com/a8m/envsubst#docs
+ENVSUBST_RELEASES := https://github.com/a8m/envsubst/releases
+ENVSUBST_VERSION := 1.2.0
+ENVSUBST_URL := $(ENVSUBST_RELEASES)/download/v$(ENVSUBST_VERSION)/envsubst-$(PLATFORM)-x86_64
+ENVSUBST := $(LOCAL_BIN)/envsubst-$(ENVSUBST_VERSION)-$(PLATFORM)-x86_64
+# We want to fail if variables are not set or empty.
+ENVSUBST_SAFE := $(ENVSUBST) -no-unset -no-empty
+$(ENVSUBST): | $(CURL) $(LOCAL_BIN)
+	$(CURL) --progress-bar --fail --location --output $(ENVSUBST) "$(ENVSUBST_URL)"
+	touch $(ENVSUBST)
+	chmod +x $(ENVSUBST)
+	ln -sf $(ENVSUBST) $(LOCAL_BIN)/envsubst
+.PHONY: envsubst
+envsubst: $(ENVSUBST)
+.PHONY: releases-envsubst
+releases-envsubst:
+	$(OPEN) $(ENVSUBST_RELEASES)
+
+KUBECTL_RELEASES := https://github.com/kubernetes/kubernetes/tags
+# Keep this in sync with KIND_K8s_VERSION
+KUBECTL_VERSION := 1.20.7
+KUBECTL_BIN := kubectl-$(KUBECTL_VERSION)-$(platform)-amd64
+KUBECTL_URL := https://storage.googleapis.com/kubernetes-release/release/v$(KUBECTL_VERSION)/bin/$(platform)/amd64/kubectl
+KUBECTL := $(LOCAL_BIN)/$(KUBECTL_BIN)
+$(KUBECTL): | $(CURL) $(LOCAL_BIN)
+	$(CURL) --progress-bar --fail --location --output $(KUBECTL) "$(KUBECTL_URL)"
+	touch $(KUBECTL)
+	chmod +x $(KUBECTL)
+	$(KUBECTL) version | grep $(KUBECTL_VERSION)
+	ln -sf $(KUBECTL) $(LOCAL_BIN)/kubectl
+.PHONY: kubectl
+kubectl: $(KUBECTL)
+.PHONY: releases-kubectl
+releases-kubectl:
+	$(OPEN) $(KUBECTL_RELEASES)
+K_CMD ?= apply
+# Dump all objects (do not apply) if DEBUG variable is set
+ifneq (,$(DEBUG))
+K_CMD = create --dry-run=client --output=yaml
+endif
+
+K9S_RELEASES := https://github.com/derailed/k9s/releases
+K9S_VERSION := 0.25.7
+K9S_BIN_DIR := $(LOCAL_BIN)/k9s-$(K9S_VERSION)-$(platform)-x86_64
+K9S_URL := $(K9S_RELEASES)/download/v$(K9S_VERSION)/k9s_$(platform)_x86_64.tar.gz
+K9S := $(K9S_BIN_DIR)/k9s
+$(K9S): | $(CURL) $(LOCAL_BIN) $(KUBECTL)
+	$(CURL) --progress-bar --fail --location --output $(K9S_BIN_DIR).tar.gz "$(K9S_URL)"
+	mkdir -p $(K9S_BIN_DIR) && tar zxf $(K9S_BIN_DIR).tar.gz -C $(K9S_BIN_DIR)
+	touch $(K9S)
+	chmod +x $(K9S)
+	$(K9S) version | grep $(K9S_VERSION)
+	ln -sf $(K9S) $(LOCAL_BIN)/k9s
+.PHONY: releases-k9s
+releases-k9s:
+	$(OPEN) $(K9S_RELEASES)
+.PHONY: k9s
+K9S_ARGS ?= --all-namespaces
+k9s: | $(KUBECONFIG) $(K9S) ## Terminal ncurses UI for K8s
+	$(K9S) $(K9S_ARGS)
+
+GH_RELEASES := https://github.com/cli/cli/releases
+GH_VERSION := 2.2.0
+GH_DIR := $(LOCAL_BIN)/gh_$(GH_VERSION)_$(platform_alt)_amd64
+GH_URL := $(GH_RELEASES)/download/v$(GH_VERSION)/$(notdir $(GH_DIR)).tar.gz
+GH := $(GH_DIR)/bin/gh
+$(GH): | $(CURL) $(LOCAL_BIN)
+	$(CURL) --progress-bar --fail --location --output $(GH_DIR).tar.gz $(GH_URL)
+	tar zxf $(GH_DIR).tar.gz -C $(LOCAL_BIN)
+	touch $(GH)
+	chmod +x $(GH)
+	$(GH) version | grep $(GH_VERSION)
+	ln -sf $(GH) $(LOCAL_BIN)/gh
+.PHONY: gh
+gh: $(GH)
+.PHONY: releases-gh
+releases-gh:
+	$(OPEN) $(GH_RELEASES)
```

Comment:
This doesn't seem to work for me
```
❯ make releases-gh
https://github.com/cli/cli/releases
bash: https://github.com/cli/cli/releases: No such file or directory
make: *** [Makefile:184: releases-gh] Error 127
```

Reply:
I don't actually know what this part of the Makefile is useful for, I don't see `$(OPEN)` defined anywhere

Reply:
https://github.com/knative-sandbox/eventing-rabbitmq/pull/525/commits/8870f7360f1c300d31d68c0f6c0e70a1eeeb864a fixes it, thanks!